CHANGELOG.md

```diff
@@ -5,7 +5,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Hammer Beta-2
+## Hammer RC-1
```

Comment:
Missing the date

Reply:
I can't put the date there yet, because we haven't released RC1 just yet. I typically backfill the release dates on later updates.